src/pages/en/guides/deploy/github.md

```diff
@@ -25,45 +25,52 @@ You can deploy an Astro site to GitHub Pages by using [GitHub Actions](https://g
     name: Github Pages Astro CI
 
     on:
-      # Trigger the workflow every time you push to the `main` branch
-      # Using a different branch name? Replace `main` with your branch’s name
+     # Trigger the workflow every time you push to the `main` branch
       push:
-        branches: [main]
+        branches: [ main ]
       # Allows you to run this workflow manually from the Actions tab on GitHub.
       workflow_dispatch:
+      
+      # Allow this job to push changes to your repository
```

Comment:
Nit: looks like this needs slightly different indentation to align with the block it refers to:

<img width="190" alt="image" src="https://user-images.githubusercontent.com/357379/182553128-a4d24844-d250-4606-b5a8-59a6ba8a33e8.png">


```suggestion
    # Allow this job to push changes to your repository
```